distribution/test/itests/test-itests-ui/src/test/java/org/codice/ddf/e2e/ITE2ERunner.java

```diff
@@ -0,0 +1,112 @@
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.e2e;
+
+import static org.codice.ddf.test.common.options.BasicOptions.recordConfiguration;
+import static org.codice.ddf.test.common.options.DebugOptions.defaultDebuggingOptions;
+import static org.codice.ddf.test.common.options.DistributionOptions.ddfDistributionOption;
+import static org.codice.ddf.test.common.options.FeatureOptions.addBootFeature;
+import static org.codice.ddf.test.common.options.LoggingOptions.defaultLogging;
+import static org.codice.ddf.test.common.options.LoggingOptions.logLevelOption;
+import static org.codice.ddf.test.common.options.PortOptions.defaultPortsOptions;
+import static org.codice.ddf.test.common.options.PortOptions.getHttpsPort;
+import static org.codice.ddf.test.common.options.TestResourcesOptions.includeTestResources;
+import static org.codice.ddf.test.common.options.VmOptions.defaultVmOptions;
+import static org.ops4j.pax.exam.CoreOptions.mavenBundle;
+import static org.ops4j.pax.exam.CoreOptions.options;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.editConfigurationFilePut;
+
+import com.github.eirslett.maven.plugins.frontend.lib.TaskExecutor;
+import java.util.HashMap;
+import java.util.Map;
+import javax.inject.Inject;
+import org.codice.ddf.sync.installer.api.SynchronizedInstaller;
+import org.codice.ddf.test.common.features.TestUtilitiesFeatures;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.options.DefaultCompositeOption;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class ITE2ERunner {
+
+  private static final String DEFAULT_YARN_PKG = "com.github.eirslett.maven.plugins.frontend.lib";
+
+  private static final String DEFAULT_LOG_LEVEL = "INFO";
+
+  private static Option yarnLoggingLevel() {
+    recordConfiguration("yarn_log_level", DEFAULT_YARN_PKG + "=" + DEFAULT_LOG_LEVEL);
+    return logLevelOption(DEFAULT_YARN_PKG, DEFAULT_LOG_LEVEL);
+  }
+
+  private static Option defaultRunnerLogging() {
+    return new DefaultCompositeOption(defaultLogging(), yarnLoggingLevel());
+  }
+
+  private static final String SYSTEM_PROPERTIES_REL_PATH = "etc/custom.system.properties";
+
+  @Configuration
+  public static Option[] examConfiguration() {
+    String parameters = System.getProperty("ace.test.parameters");
```

Comment:
✏️ Could use `System.getProperty("ace.test.parameters", "")` to automatically handle the default value.